taui/src/components/form.js

```diff
@@ -49,7 +49,8 @@ export default class Form extends React.PureComponent {
       destination,
       network: networks && networks.length ? {
         label: networks[0].name, value: networks[0].url
-      } : null
+      } : null,
+      useNonECC: true
```

Comment:
I might be missing something that happened during later conversations, but my impression from the initial email on this feature was that this should be unchecked by default so that we nudge users toward the expanded choice communities by only showing those by default. Feel free to disregard if that has changed or I'm misunderstanding.